### ISSUE TITLE
Dependency incompatibility prevents use of app-simulator

### ISSUE THREAD

**Issue by pcherna** [onsigntv/app-simulator#1]
Running **onsigntv-app-simulator** fails:

```
❯ onsigntv-app-simulator
Traceback (most recent call last):
  File "/Users/peter/dev/x/venv/bin/onsigntv-app-simulator", line 5, in <module>
    from app_simulator.__main__ import main
  File "/Users/peter/dev/x/venv/lib/python3.9/site-packages/app_simulator/__main__.py", line 8, in <module>
    from .routes import (
  File "/Users/peter/dev/x/venv/lib/python3.9/site-packages/app_simulator/routes.py", line 8, in <module>
    from jinja2 import Environment, PackageLoader, select_autoescape
  File "/Users/peter/dev/_x/venv/lib/python3.9/site-packages/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/Users/peter/dev/x/venv/lib/python3.9/site-packages/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/Users/peter/dev/x/venv/lib/python3.9/site-packages/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
  File "/Users/peter/dev/x/venv/lib/python3.9/site-packages/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/Users/peter/dev/x/venv/lib/python3.9/site-packages/markupsafe/__init__.py)
```

According to https://github.com/pallets/markupsafe/issues/282 using Jinja 2.11.x is no longer supported. One solution is just pin the version of markupsafe at < 2.1, otherwise update Jinja2 and test it.

**Comment by pcherna** [comment]
As a workaround, the following seems to work:

```
pip install markupsafe==2.0.1
```

**Comment by edufelipe** [comment]
Hi @pcherna!

Thanks for reporting this issue. We released a new version of the app simulator, 0.9.2, that fixes this dependency issue.

Hope it works for you!
